spec/services/procedure_archive_service_spec.rb

```diff
@@ -128,10 +128,14 @@
       let(:archive) { create(:archive, time_span_type: 'monthly', status: 'pending', month: date_month) }
       let(:year) { 2021 }
       let(:mailer) { double('mailer', deliver_later: true) }
-
+      before do
+        allow_any_instance_of(ActiveStorage::Attached::One).to receive(:url).and_return("http://file.to/get.ext")
+      end
       it 'collect files' do
         expect(InstructeurMailer).to receive(:send_archive).and_return(mailer)
-        service.collect_files_archive(archive, instructeur)
+        VCR.use_cassette('archive/file.to.get') do
```

Comment:
Le fichier de cassette s'appelle `file_to_get`, mais ici il est chargé avec `file.to.get`. C'est voulu ?

Reply:
nop